static/src/stylesheets/email/_front.scss

```diff
@@ -155,17 +154,16 @@ $container-color: #ffffff;
     padding: 3px 5px;
```

Comment:
not part of these changes, but I'd double-check that the `margin-left` and `display: inline-block` on `.review-stars` have the intended effect across clients

Reply:
Yep that's bad. I'll tackle this in a seperate PR this afternoon.